Library/Formula/wartremover.rb

```diff
@@ -0,0 +1,31 @@
+class Wartremover < Formula
+  desc "Flexible Scala code linting tool"
+  homepage "https://github.com/puffnfresh/wartremover"
+  url "https://github.com/puffnfresh/wartremover/archive/v0.13.tar.gz"
+  sha256 "65d2f9907662bc2cc892801055713dd80a7fc25ade5091cb534bfc51eaae626b"
+
+  head "https://github.com/puffnfresh/wartremover.git"
+
+  depends_on "sbt" => :build
+
+  def install
+    system "sbt", "core/assembly"
+    libexec.install Dir["core/target/scala-*/wartremover-assembly-*.jar"]
+    bin.write_jar_script Dir[libexec/"wartremover-assembly-*.jar"][0], "wartremover"
```

Comment:
`Dir[pattern]` returns a collection, and `write_jar_script` expects a single `Path` as its first argument. The globbing is necessary because we don't know what version number will replace the `*`.